nym-vpn-apple/Scripts/UpdateDaemonInfoPlist.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
```

Comment:
I'd recommend adding a comment block here at the top containing example invocations. I think this is supposed to be called from the `nym-vpn-apple/Scripts` directory with the version as argument?

```sh
# Example:
#   nym-vpn-apple/Scripts$ ./Scripts/UpdateDaemonInfoPlist.sh 0.1.1
#
```